confluent/docker_utils/cub.py

```diff
@@ -407,11 +428,15 @@ def main():
     sr.add_argument('host', help='Hostname for Schema Registry.')
     sr.add_argument('port', help='Port for Schema Registry.')
     sr.add_argument('timeout', help='Time in secs to wait for service to be ready.', type=int)
+    sr.add_argument('--secure', help='Use TLS to secure the connection.', action='store_true')
+    sr.add_argument('--ignore_cert', help='Ignore TLS certificate errors.', action='store_true')
```

Comment:
https://docs.python.org/3/library/argparse.html#action

```
'store_true' and 'store_false' - These are special cases of 'store_const' used for storing the values True and False respectively. In addition, they create default values of False and True respectively. For example:
```

So by default 
* secure=false (Use `http`) 👍 not changing behavior
* ignore_cert=false (Don't ignore, enforce good PKI practices) 👍 on good default behavior.